src/node/stage/approved.rs

```diff
@@ -2431,9 +2414,16 @@ impl Approved {
                 VoteTheirKnowledge { prefix, key_index } => {
                     self.cast_unordered_vote(core, Vote::TheirKnowledge { prefix, key_index })?
                 }
-                VoteSendNeighbourInfo { dst, nonce } => {
-                    self.cast_ordered_vote(AccumulatingEvent::SendNeighbourInfo { dst, nonce })
-                }
+                SendNeighbourInfo { dst, nonce } => self.send_routing_message(
+                    core,
+                    SrcLocation::Section(*self.shared_state.our_prefix()),
+                    DstLocation::Section(dst),
+                    Variant::NeighbourInfo {
+                        elders_info: self.shared_state.our_info().clone(),
+                        nonce,
+                    },
+                    None,
+                )?,
```

Comment:
I think we can do better than this. The elders info of our section is already signed, so we don't need to re-accumulate the messages. We can just send it as `Node` -> `Section` message. Just need to change the `NeighbourInfo` variant to contain `Proven<EldersInfo>` instead of just `EldersInfo`. Also we need to attach a sufficient proof to the message and implement the proof verification for this variant (see `Variant::verify`).

Reply:
Ok, will have a try with the change